redaxo/src/core/lib/console/setup/run.php

```diff
@@ -0,0 +1,424 @@
+        }
+
+        if (!$input->getOption('timezone')) {
+            $q = new Question('Choose timezone', $config['timezone']);
+            $q->setAutocompleterValues(DateTimeZone::listIdentifiers());
+            $q->setValidator(static function ($value) {
+                if (false === @date_default_timezone_set($value)) {
+                    throw new RuntimeException('Time zone invalid');
+                }
+                return $value;
+            });
+            $config['timezone'] = $io->askQuestion($q);
+        } else {
+            $timezone = $input->getOption('timezone');
+            if (!in_array($timezone, DateTimeZone::listIdentifiers(), true)) {
+                throw new InvalidArgumentException('Unknown timezone "'.$timezone.'" specified');
+            }
+            $config['timezone'] = $timezone;
+            $io->success('Timezone "'.$timezone.'" selected');
+        }
+
+        if (!$input->getOption('db-host') && !$input->getOption('db-login') && !$input->getOption('db-password') && !$input->getOption('db-name')) {
+            $io->section('Database information');
+            do {
+                $config['db'][1]['host'] = $io->ask('MySQL host', $config['db'][1]['host']);
+                $config['db'][1]['login'] = $io->ask('Login', $config['db'][1]['login']);
+                $config['db'][1]['password'] = $io->askHidden('Password');
+                $config['db'][1]['name'] = $io->ask('Database name', $config['db'][1]['name']);
+
+                $redaxo_db_create = $io->confirm('Create database', false);
+
+                rex::setProperty('db', $config['db']);
+                try {
+                    $err = rex_setup::checkDb($config, $redaxo_db_create);
+                } catch (PDOException $e) {
+                    $err = 'The following error occured: ' . $e->getMessage();
+                }
+
+                if ('' !== $err) {
+                    $io->error($err);
+                }
+            } while ('' !== $err);
+        } else {
+            $config['db'][1]['host'] = $input->getOption('db-host');
+            $config['db'][1]['login'] = $input->getOption('db-login');
+            $config['db'][1]['password'] = $input->getOption('db-password');
+            $config['db'][1]['name'] = $input->getOption('db-name');
+
+            $redaxo_db_create = true === $input->getOption('db-createdb');
+
+            rex::setProperty('db', $config['db']);
+            try {
+                $err = rex_setup::checkDb($config, $redaxo_db_create);
+            } catch (PDOException $e) {
+                $err = 'The following error occured: ' . $e->getMessage();
+            }
+
+            if ('' !== $err) {
+                $io->error($err);
+                return 1;
+            }
+        }
+        $io->success('Database connection successfully established');
+
+        // ---------------------------------- step 5 . create db / demo
+        $io->title('Step 5 of 6 / Database');
```

Comment:
Im Backend gebe ich bei diesem Schritt ja oben die Datenbank-Version aus. (siehe screenshot hier: https://github.com/redaxo/redaxo/pull/3150)
Glaube, das fände ich in der Console auch ganz gut.

Reply:
Oder ggf. in der Erfolgsmeldung im Schritt zuvor.
Aber so, dass man irgendwo nochmal sieht, zu was für einer DB-Version man sich nun verbunden hat.